### PR TITLE
Call setReady in Netlify

### PR DESCRIPTION
**Pull request by giotiskl**

### CODE REVIEW

File: marketplace/netlify-build/src/app/index.js [hunk 1]
```diff
@@ -64,14 +64,17 @@ export default class NetlifyAppConfig extends React.Component {
...
-      ready: true,
-      config,
-      enabledContentTypes,
-      contentTypes: contentTypesResponse.items.map(ct => [ct.sys.id, ct.name]),
-      netlifySites: uniqBy(netlifySites, s => s.id),
-      ticketId
-    });
+    this.setState(
+      {
+        ready: true,
```
Comment (jelz) [inline marketplace/netlify-build/src/app/index.js]: Remove this and remove all code related to it

File: marketplace/netlify-build/src/app/index.js [hunk 2]
```diff
@@ -64,14 +64,17 @@ export default class NetlifyAppConfig extends React.Component {
...
+        config,
+        enabledContentTypes,
+        contentTypes: contentTypesResponse.items.map(ct => [ct.sys.id, ct.name]),
+        netlifySites: uniqBy(netlifySites, s => s.id),
+        ticketId
+      },
+      app.setReady
```
Comment (jelz) [inline marketplace/netlify-build/src/app/index.js]: I'd wrap with a function

File: marketplace/netlify-build/src/app/index.js (hunk 1 above)
Comment (giotiskl) [reply marketplace/netlify-build/src/app/index.js]: done

File: marketplace/netlify-build/src/app/index.js (hunk 2 above)
Comment (giotiskl) [reply marketplace/netlify-build/src/app/index.js]: any reason to go for the more verbose variation?

Comment (jelz) [reply marketplace/netlify-build/src/app/index.js]: so we make sure it's called with 0 args (just to be safe)

File: marketplace/netlify-build/src/app/index.js [hunk 3]
```diff
@@ -171,39 +177,33 @@ export default class NetlifyAppConfig extends React.Component {
 
   render() {
     const disabled = !this.state.token;
+    console.log('config', this.state.config);
```
Comment (jelz) [inline marketplace/netlify-build/src/app/index.js]: delete

**Pull request merged**